Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -3887,6 +3887,54 @@ class ConstructCircumcenter_3 {
 
 }; /* end Kernel::ConstructCircumcenter_3 */
 
+
+/*!
+  \ingroup PkgKernel23ConceptsFunctionObjects
+  \cgalConcept
+
+  \cgalRefines `AdaptableFunctor` (with four arguments) 
+
+  \sa `closest_point_grp`
+
+*/
+class ConstructClosestPoint_3 {
+public:
+
+  /// \name Operations
+  /// A model of this concept must provide:
+  /// @{
+
+   /**
+     * @brief Computes the closest_point from query between bound and
+     * any point of segment.
+     * @param query the query point
+     * @param segment the segment
+     * @param bound the farthest point
+     * @param k the kernel
+     * @return nearest point: bound or a point inside segment
+     */
+  Kernel::Point_3 operator()(const Kernel::Point_3& query,
+                             const Kernel::Segment_3& segment,
+                             const  Kernel::Point_3& bound);
+
+  /**
+     * @brief Computes the closest_point from query between bound and
+     * any point of triangle.
+     * @param query the query point
+     * @param triangle the triangle
+     * @param bound the farthest point
+     * @param k the kernel
+     * @return nearest point: bound or a point inside triangle
+     */
```

Comment:
todo : add backquotes to variables names in the documentation, to get the right layout